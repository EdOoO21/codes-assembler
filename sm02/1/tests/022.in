-504 -952 255 -85 -623 437 -754 514 334 799 -293 623 821 -837 -526 -449 965
