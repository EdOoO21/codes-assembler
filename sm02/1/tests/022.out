965
-449
-526
-837
821
623
-293
799
334
514
-754
437
-623
-85
255
-952
-504
